roles/redhat_satellite6_capsule_lb/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for satellite_lb
+domain: example.com
+sat_server: sat6-master-01.{{ domain }}
+capsule1: sat6-capsule-01.{{ domain }}
```

Comment:
shoulnd't we just use the hosts listed in like a `satellite_capsules` hostgroup that have like a `lb_group=FOO` variable on them? or if we don't want to get so fancy, just a `satellite_content_capsules` group and if a `lb_fqdn` is set then lb all the hosts in that group?